tests/simulator/test_advance.py

```diff
@@ -289,10 +289,28 @@ def base_test_overlaped_fields_are_equal(self, datahier, coarsest_time):
                         # seems correct considering ghosts are filled with schedules
                         # involving linear/spatial interpolations and so on where
                         # rounding errors may occur.... setting atol to 5.5e-15
-                        assert_fp_any_all_close(slice1, slice2, atol=5.5e-15, rtol=0)
+                        assert_fp_any_all_close(slice1, slice2, atol=2.5e-14, rtol=0)
```

Comment:
do you remember why this was changed

Reply:
the proposed value is not consistent with the comment that has been changed accordingly. I think it's nice if we can keep track of why we raise these bars by providing the build that failed for previous threshold

Reply:
my bad... I dont remember why it went so,

Reply:
so it means that it fails with 5.5e-15?